test/autoCompletionFix.test.ts

```diff
@@ -538,4 +538,47 @@ objB:
     expect(completion.items.length).equal(1);
     expect(completion.items[0].insertText).to.be.equal('test1');
   });
+  
+  describe('should suggest property before indented comment', () => {
+    const schema: JSONSchema = {
+      type: 'object',
+      properties: {
+        example: {
+          type: 'object',
+          properties: {
+            prop1: {
+              type: 'string',
+            },
+            prop2: {
+              type: 'string',
+            },
+          },
+        },
+      }
+    };
+
+    it('completion should handle indented comment on new line', async () => {
+      languageService.addSchema(SCHEMA_ID, schema);
+      const content = 'example:\n  prop1: "test"\n  \n    #comment';
+      const completion = await parseSetup(content, 2, 2);
+      expect(completion.items.length).equal(1);
+      expect(completion.items[0]).to.be.deep.equal(
+        createExpectedCompletion('prop2', 'prop2: ', 2, 2, 2, 2, CompletionItemKind.Property, InsertTextFormat.Snippet, {
+          documentation: ''
+        })
+      );
+    });
+
+    it('completion should handle comment at same indent level on new line', async () => {
```

Comment:
This unit test is testing functionality that was already working before the fix. Simply added as a control check.